CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ Changelog
 
 ### Chores
 
+- Updated newsfeed ([PR 1786](https://github.com/input-output-hk/daedalus/pull/1786))
```

Comment:
Shouldn't this be under Features?

Reply:
@daniloprates really not sure can be both. But we already have newsfeed feature this is just update for current feature so I decided to put that in `Chore`. Let me know if you think that I'm in wrong and if is better to put that under Features.